docker/bungeecord/removal-username-history-api-20220913.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/src/main/java/com/imaginarycode/minecraft/redisbungee/util/uuid/NameFetcher.java b/src/main/java/com/imaginarycode/minecraft/redisbungee/util/uuid/NameFetcher.java
+index b7c035f..e2e5c38 100644
+--- a/src/main/java/com/imaginarycode/minecraft/redisbungee/util/uuid/NameFetcher.java
++++ b/src/main/java/com/imaginarycode/minecraft/redisbungee/util/uuid/NameFetcher.java
+@@ -12,6 +12,7 @@ import lombok.Setter;
+ import java.io.IOException;
+ import java.lang.reflect.Type;
+ import java.util.ArrayList;
++import java.util.Arrays;
+ import java.util.List;
+ import java.util.UUID;
+ 
+@@ -20,7 +21,15 @@ public class NameFetcher {
+     @Setter
+     private static OkHttpClient httpClient;
+ 
++    /**
++     * The API https://api.mojang.com/user/profiles/ has removed at 13 SEP 2022.
++     * So now this method doesn't access the API, but returns empty list.
++     * @param uuid
++     * @return Empty List
++     * @throws IOException
```

Comment:
```suggestion
+     * Note: This method has been patched.
+     * This method is used to get user's IGN history from https://api.mojang.com/user/profiles/ , however Mojang removed public access in 13 Sep 2022. Please see https://help.minecraft.net/hc/en-us/articles/8969841895693-Username-History-API-Removal-FAQ- for more info.
+     * @param uuid the target player
+     * @return Injected list. returns non-empty list for UUID, which is given in test. otherwise, returns empty list.
+     * @throws IOException
```

@kory33 こんなかんじでどうですか